wikipedia-dark.user.css

```diff
@@ -245,8 +245,8 @@ domain("wiki.rpcs3.net") {
   #mp-left h2, #mp-left h2 span, td[style*="background: #99FF99;" i], .table-yes,
   td[style*="background:#dfd" i], td[style*="background:#bfd" i],
   tr[style*="background:#DDFFDD" i], tr[style*="background:#bfb" i],
-  tr[style*="background-color:#CCFFCC" i] {
-    background: #244024 !important;
+  tr[style*="background-color:#CCFFCC" i], .tpl-infobox-header {
+    background: #006600 !important;
```

Comment:
Making the green brighter is fine. Please change it to use the shorthand notation (`#060`).

Do you think any of the other colors need to be brighter?

Reply:
Actually, since you asked I checked out the [different type of headers](https://www.mediawiki.org/wiki/Extension_status) and my code turns all headers to green. So this is gonna require some additional work. From a quick glance of all the headers, I'd say only green and grey are hard to read. Rest all seem to work well with our theme. Will push a proper fix for both soon.